### DISCUSSION TITLE
Support for Snowflake

### DISCUSSION THREAD

**Discussion by debigithub** [dineug/erd-editor#383] [General]
Hi - Is there any plan for Snowflake support? 
Sorry if this is not the right place - couldn't find any specific links for feature requests.

**Comment by dineug** [comment]
You can register it as an issue.

I’m not familiar with Snowflake—what specific support is needed?

**Reply by debigithub** [reply]
Sorry its not specific to any issue but a feature request. I am trying to generate SQL which is compatible to Snowflake database- similar to how it is doing postgres,MSSQL etc. I have attached screenshot.
![screenshot](https://github.com/user-attachments/assets/c2fa7367-ed0b-4eaa-90c1-f720d9b5fc48)

**Reply by dineug** [reply]
You can submit a feature request for adding support for a specific database vendor.  
Here’s the link to register your issue:  
[https://github.com/dineug/erd-editor/issues/new/choose](https://github.com/dineug/erd-editor/issues/new/choose)

**Comment by dineug** [comment]
#435
